metricbeat/module/docker/glide.yaml

```diff
@@ -2,3 +2,6 @@ package: github.com/elastic/beats/metricbeat/module/docker
 import:
 - package: github.com/fsouza/go-dockerclient
   version: 23c589186c2a92a8742da55f19aec4997dae5cbb
+import:
```

Comment:
Comparing these two commits, there are many more changes: https://github.com/fsouza/go-dockerclient/compare/23c589186c2a92a8742da55f19aec4997dae5cbb...45c1a814bcd9b656bdb9813a7c21d0236fc301db I will update the library in a separate commit so you don't have to update it yourself.